### DISCUSSION TITLE
What can I do better for F# Weekly in 2021?

### DISCUSSION THREAD

**Discussion by sergey-tihon** [sergey-tihon/fsharp-weekly#1] [Q&A]
If you have an idea how to improve F# Weekly - please share
- Do I have section that you always skip?
- Do I miss some important news or update?
- Other ideas or recommendations?

For some reason people downvote weekly editions in 2021 😢 
<img width="695" alt="image" src="https://user-images.githubusercontent.com/1197905/106437687-cbea1900-6475-11eb-94d4-7a3ac5c83bb0.png">
<img width="695" alt="image" src="https://user-images.githubusercontent.com/1197905/106437590-b4ab2b80-6475-11eb-91d4-14d554da16cf.png">
<img width="695" alt="image" src="https://user-images.githubusercontent.com/1197905/106437626-becd2a00-6475-11eb-84ed-93d63fc699ed.png">
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1197905/106462203-0f547f80-6496-11eb-8d42-231a63f980c4.png">

**Comment by do-wa** [comment]
I appreciate your effort in doing this and don't understand the downvotes. Especially in this small community a source of consistent weekly news is important to discover new libs, language news, videos and any sort of updates to at least convey the feeling of a living community.

To give you any feedback:
I like the layout with categories and just links to the resources. The scattered tweets in between are (at least for me) sometimes difficult to "parse" in the overall context. Maybe they can be visually separated from the content or my 1080p horizontal space can be used better. (This is feedback from a desktop only visitor who got here from a dsyme tweet .. maybe those tweets are not that bad to discover other community members 😉  )

Keep up the good work

**Comment by 2mol** [comment]
I've never seen any option for giving stars/voting, is that on Wordpress?

**Reply by sergey-tihon** [reply]
Yes, it is a WordPress feature, I see it at the top of [every post](https://sergeytihon.com/2021/01/30/f-weekly-5-2021-f-friendly-nuget-org/)

**Comment by akhansari** [comment]
I didn't even know there was a voting system because I often consult the email version.
It's my weekly ritual to read it and I appreciate too your effort.
Keep up the great work.

**Comment by Krzysztof-Cieslak** [comment]
I'd remove the ability to vote. The content is great, and the work is thankless (BTW, thank you! ❤), it's not worth to worry about some non-authenticated, anonymous voting on the internet.

**Reply by sergey-tihon** [reply]
HA! BTW, good idea, as usual =)

**Comment by beyon** [comment]
I have a few minor things I think could be improved:

At least on mobile the content often moves around after a while which has caused me to click the wrong link multiple times. I guess some element of the page changes size when it loads so they whole layout changes.

In the "Videos and Slides" section entries could be tagged with something like [video]/[slides]/[pdf]/[podcast] so you know what will load if you press them. Again mostly an issue on the phone were I can't hover to see the actual link.

Other than that it's great resource that I appreciate that you are doing it.
